### PR TITLE
 JDP230601-5 

### PR DESCRIPTION
**Pull request by ZuzannJanik**
### Jira ticket:  JDP230601-5 
https://projects.kodilla.com/secure/RapidBoard.jspa?rapidView=217&projectKey=JDP230601&view=detail&selectedIssue=JDP230601-5

### What has changed:
- Changing dependencies permanently from fixed with appointments
- Introduction of package structure
- Initialization of the UserController class

### What reviewers should know:
nothing

### CODE REVIEW

**Review by piotr-grosicki — CHANGES_REQUESTED** [review]
projekt sie nie buduje

File: build.gradle [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
 
 repositories {
     mavenCentral()
 }
 
-
 dependencies {
-    implementation('org.springframework.boot:spring-boot-starter-data-jpa')
-    implementation('org.springframework.boot:spring-boot-starter-web')
-    runtimeOnly('com.h2database:h2')
```
Comment (piotr-grosicki) [inline build.gradle:30]: h2 musi zostac, nie moze zostac usunieta poniewaz testy z niej korzystaja

File: src/main/java/com/kodilla/ecommercee/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    @GetMapping
+    public List<UserDto> getUsers() {
+        return new ArrayList<>();
+    }
+    @GetMapping(value = "{userId}")
+    public UserDto getUser(@PathVariable Long orderId) {
+        return new UserDto(1L);
+    }
+    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    public void createUser(UserDto userDto) {
```
Comment (piotr-grosicki) [inline src/main/java/com/kodilla/ecommercee/controller/UserController.java:25]: RequestBody

File: src/main/java/com/kodilla/ecommercee/controller/UserController.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    }
+    @PutMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    public UserDto blockUser(UserDto userDto) {
```
Comment (piotr-grosicki) [inline src/main/java/com/kodilla/ecommercee/controller/UserController.java:28]: RequestBody

File: src/main/java/com/kodilla/ecommercee/controller/UserController.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        return new UserDto(1L);
+    }
+    @PutMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    public void generateToken(UserDto userDto) {
```
Comment (piotr-grosicki) [inline src/main/java/com/kodilla/ecommercee/controller/UserController.java:32]: RequestBody

File: src/main/java/com/kodilla/ecommercee/controller/UserController.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    }
+    @DeleteMapping
+    public  void deleteUser(Long userId) {
```
Comment (piotr-grosicki) [inline src/main/java/com/kodilla/ecommercee/controller/UserController.java:35]: PathVariable

**Review by piotr-grosicki — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (ZuzannJanik) [reply build.gradle:30]: ok

File: src/main/java/com/kodilla/ecommercee/controller/UserController.java (hunk 2 above)
Comment (ZuzannJanik) [reply src/main/java/com/kodilla/ecommercee/controller/UserController.java:25]: ok

File: src/main/java/com/kodilla/ecommercee/controller/UserController.java (hunk 3 above)
Comment (ZuzannJanik) [reply src/main/java/com/kodilla/ecommercee/controller/UserController.java:28]: ok

File: src/main/java/com/kodilla/ecommercee/controller/UserController.java (hunk 4 above)
Comment (ZuzannJanik) [reply src/main/java/com/kodilla/ecommercee/controller/UserController.java:32]: ok

File: src/main/java/com/kodilla/ecommercee/controller/UserController.java (hunk 5 above)
Comment (ZuzannJanik) [reply src/main/java/com/kodilla/ecommercee/controller/UserController.java:35]: ok

**Pull request merged**